ui/pages/confirmations/components/confirm/title/title.tsx

```diff
@@ -203,7 +204,13 @@ const getDescription = (
         return t('confirmTitleDescPermitSignature');
       }
       if ((confirmation as SignatureRequestType).decodedPermission) {
-        return t('confirmTitleDescPermission');
+        const permissionType = (confirmation as SignatureRequestType)
+          .decodedPermission?.permission?.type;
+
+        return getPermissionDescription(
+          t as ReturnType<typeof useI18nContext>,
```

Comment:
`t` is already typed as `IntlFunction`. Should we redefine `IntlFunction` as an alias of `ReturnType<typeof useI18nContext>` to keep things consistent?